openid4java/1.0.0.wso2v6/pom.xml

```diff
@@ -0,0 +1,192 @@
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This is old license. Please use the new one [1]

[1] https://github.com/wso2/orbit/pull/694/files

Reply:
changed the license